src/extensions/utils.ts

```diff
@@ -33,12 +33,14 @@ export function convertExtensionOptionToLabeledList(options: ParamOption[]): Lis
 export function convertOfficialExtensionsToList(officialExts: {
   [key: string]: RegistryEntry;
 }): ListItem[] {
-  return _.map(officialExts, (entry: RegistryEntry, key: string) => {
+  const l = _.map(officialExts, (entry: RegistryEntry, key: string) => {
```

Comment:
Nit: Prefer using native functions (`officialExts.map(...).sort(...)`) instead of lodash

Reply:
officialExts is an obj so there's no native map, could do object.entries but since this code will get deleted soon I'm leaving it as is